compass/sign/RemoteSignatureSource.java

```diff
@@ -40,15 +48,67 @@ public RemoteSignatureSource(String uri,
                                String trustCertCollectionFilePath,
                                String clientCertChainFilePath,
                                String clientPrivateKeyFilePath) throws SSLException {
-    this(NettyChannelBuilder
+    this(OkHttpChannelBuilder
         .forTarget(uri)
         .useTransportSecurity()
         .idleTimeout(5, TimeUnit.SECONDS)
-        .sslContext(
-            buildSslContext(trustCertCollectionFilePath, clientCertChainFilePath, clientPrivateKeyFilePath))
+        .sslSocketFactory(createClientCertSslSocketFactory(trustCertCollectionFilePath, clientCertChainFilePath, clientPrivateKeyFilePath))
         .build());
   }
 
+  private static PrivateKey createPrivateKeyFromPemFile(final String keyFileName) throws IOException, InvalidKeySpecException, NoSuchAlgorithmException {
+      byte []pemContent;
```

Comment:
```suggestion
      byte[] pemContent;
```

It doesn't really matter, but it just makes me uneasy that the space is in that position